azure-devops/azext_devops/dev/common/const.py

```diff
@@ -24,3 +24,35 @@
 DEVOPS_TEAM_PROJECT_DEFAULT = 'project'
 PAT_ENV_VARIABLE_NAME = CLI_ENV_VARIABLE_PREFIX + 'PAT'
 AUTH_TOKEN_ENV_VARIABLE_NAME = CLI_ENV_VARIABLE_PREFIX + 'AUTH_TOKEN'
+
+# policy constants
```

Comment:
This seems like unintended change introduced with resolving merge conflict? Policy constants section should be removed from here.

Reply:
Will do - yeah - I rebased onto master before pushing from memory so maybe I introduced that then.